### DISCUSSION TITLE
VPN with port forward

### DISCUSSION THREAD

**Discussion by karserasl** [qdm12/gluetun#1642] [Q&A]
Now that Mullvad deleted their port forward, which one do you guys recommend?

i found Airvpn, IVPN, Proton ?

**Comment by VonPuffelen** [comment]
Too bad, just now that I have taken out a year's subscription with Mullvad.  Still having some boot issues with them. I'm going to rely on their 30-day money-back guarantee. I used to have AirVPN but you can't pay cash there. Now back to AirVPN

**Reply by karserasl** [reply]
Cancelled and asked for refund from Mullvad.
Went with AirVPN today. It seems atleast equivalent with Mullvad. Setup Wireguard (getting >800Mbits on some servers with my 1Gbit connection). 
Port forwarding is working and you can check it from their webapp (Mullvad you had to exec into the container and curl a website)

**Reply by VonPuffelen** [reply]
When you pay cash you don't have a 30 day money back guarantee I just read. Too bad. I'm going back to AirVPN. This worked well with port forwarding.

**Comment by vuanhson** [comment]
Luckily I subscribed Mullvad month by month, now I'm using ProtonVPN. My main use case is torrenting with port forwarding and it still very good.
